src/Analyzer/Passes/QueryAnalysisPass.cpp

```diff
@@ -2854,6 +2854,8 @@ IdentifierResolveResult QueryAnalyzer::tryResolveIdentifierInParentScopes(const
         }
     }
 
+    identifier_resolve_settings.allow_to_check_join_tree = false;
```

Comment:
This line definitely requires a comment with an explanation.

Reply:
Updated.